### PR TITLE
feat: Expose annotations for structs and services

### PR DESCRIPTION
**Pull request by kevin-greene-ck**

### CODE REVIEW

**Comment by kevin-greene-ck** [comment]
@rbattala-ck I know you're probably not that familiar with this code yet, but if you could just give this a sanity check it would be much appreciated. I need to update some of the test fixtures to have empty annotations, but the logic should be ready.

File: src/tests/unit/fixtures/thrift-server/annotations_struct.solution.ts [hunk 1]
```diff
@@ -0,0 +1,112 @@
...
+                bigID: (_args.bigID != null ? _args.bigID : thrift.Int64.fromDecimalString("23948234"))
+            };
+        }
+        else {
+            throw new thrift.TProtocolException(thrift.TProtocolExceptionType.UNKNOWN, "Unable to read MyStruct from input");
+        }
+    }
+};
+export class MyStruct extends thrift.StructLike implements IMyStruct {
+    public id: number = 45;
```
Comment (rbattala-ck) [inline src/tests/unit/fixtures/thrift-server/annotations_struct.solution.ts]: why did we assign default values to MyStruct?

Comment (kevin-greene-ck) [reply src/tests/unit/fixtures/thrift-server/annotations_struct.solution.ts]: These are the user-defined defaults. We set them here and override them in the constructor if they pass in other values.

**Comment by kevin-greene-ck** [comment]
This isn't handling services that extend other services yet. A service should inherit annotations.
